plugins/main/public/components/agents/syscollector/components/network-interfaces-table.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { EuiPanel } from '@elastic/eui';
+import { SEARCH_BAR_WQL_VALUE_SUGGESTIONS_COUNT } from '../../../../../common/constants';
+import { TableWzAPI } from '../../../common/tables';
+import { WzRequest } from '../../../../react-services';
+import { get as getLodash } from 'lodash';
+import { netifaceColumns } from '../columns';
+
+const sortFieldSuggestion = (a, b) => (a.label > b.label ? 1 : -1);
+
+export const NetworkInterfacesTable = ({ agent }) => {
+  return (
+    <EuiPanel paddingSize='m' style={{ margin: '12px 16px 12px 16px' }}>
```

Comment:
**nitpick**: this panel component has different properties of the rest syscollector panels. Is there a reason for this or only forget to replace it?

Reply:
Some tables need the `soPlatform` property to determine which column will render, but some tables don't need this property as it does not depend on the operating system to render the information.